_layouts/default.html

```diff
@@ -6,17 +6,38 @@
         <header>
             <nav class="ui secondary menu">
                 <div class="right menu">
-                    {% if page.url != "/" and page.url != "/en/" %}
+                    {% if page.ref != 'landing' %}
                         <a class="item" id="logo" href="/">beta.gouv.fr</a>
                     {% endif %}
 
                     {% assign menu_items = site.pages | where_exp: 'page', 'page.menu_index > 0' | sort: 'menu_index' %}
```

Comment:
This struck me as hard to understand, it took me several tries to get it. There seem to be contradictory assumptions: in displaying titles English is assumed to be the only language we have translations for (at least that's how I read it), whereas the "switches" portion allows for more than two languages.

In any case this is probably worth extracting to a separate file now.